packaging/docker/Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN apk add --no-cache ca-certificates bash tzdata && \
 
 # Oracle Support for x86_64 only
 RUN if [ `arch` = "x86_64" ]; then \
-      apk add --no-cache --upgrade --repository=http://dl-cdn.alpinelinux.org/alpine/edge/community libaio libnsl && \
+      apk add --no-cache --upgrade libaio libnsl && \
```

Comment:
Need to verify with @briangann this doesn't break Oracle plugin support

Reply:
@marefr Heard from @briangann that it shouldn't be a problem.